tests/test_bundle.py

```diff
@@ -85,24 +85,26 @@ def _test_bundle_get(self, replica: Replica):
 
     @testmode.integration
     def test_bundle_get_directaccess(self):
```

Comment:
This test parses out they key from the direct url and verifies contents.

Is there a test somewhere that confirms the direct url points to the file we expect?

Reply:
Contents should be good enough.  It does verify that the bucket is the checkout bucket.

Reply:
https://github.com/HumanCellAtlas/data-store/pull/1545/files#diff-307f004e5f5806557cc669e52b0cdd1eR126

Reply:
Ok, one more question and then I'll hold my peace:  Does it verify that `aws s3 cp {direct_url} .` will work?

Reply:
No, it does not.